.github/workflows/build.yml

```diff
@@ -0,0 +1,136 @@
+# Builds the Betaflight firmware
+#
+# After building, artifacts are released to a seperate repository.
```

Comment:
I know is copy& paste, but separate and not seperate

Reply:
LOL I just gonna raise a typo fix PR for those and this make me do less job!